_pages/dev/features/launch-dialog.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Launch Dialog Service
+---
+
+The `LaunchDialogService` facilitates the opening and rendering of dialog components within your storefront application. It provides methods for launching dialogs, managing the dialog lifecycle, and passing data to the rendered components.
+
+Before using the `LaunchDialogService`, you need to import it into your component or service, as shown in the following example:
+
+```typescript
+import { Injectable } from '@angular/core';
+import { LaunchDialogService } from '@spartacus/storefront';
+
+@Injectable()
+export class YourService {
+  constructor(protected launchDialogService: LaunchDialogService) {
+    // ...
+  }
+}
+```
+
+## Launch Dialog Service Properties
+
+The `LaunchDialogService` exposes the following properties:
+
+- `data$`, which is an Observable that provides access to data updates.
+- `dialogClose`, which is an Observable that provides access to dialog close events.
+
```

Comment:
I noticed in `lauch-dialog-service.ts` that there is a `_dataSubject` that is defined just below `_dialogClose`. This led me to wonder if this is also a property of the `LaunchDialogService`, and if so, do we need to mention it in the documentation? 🙂

Reply:
You can see that I mentioned in the documentation `dialogClose` and not` _dialogClose`. This is because `_dialogClose` is a private property of LaunchDialogService and is not accessible from outside. To access data from this property, you should use the dialogClose property, which is explicitly mentioned in the documentation.

The same story with `_dataSubject`, it is private, so in documentation was mentioned `data$`.
`data$`, which is an Observable that provides access to data updates.

It is designed this way to control modifications of these properties. This means they can be modified only in LaunchDialogService, and outside of it, the only possibility is to access the data using the provided `getters`.